packages/sandcastle/index.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8" />
     <link rel="icon" type="image/svg+xml" href="/favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Sandcastle Reborn</title>
+    <title>Cesium Sandcastle</title>
```

Comment:
```suggestion
    <title>CesiumJS Sandcastle</title>
```

I think, technically speaking, this should be the branding?

Reply:
Possibly? I was just matching what the current Sandcastle uses